* make sure we always look for tools in the path, no "do it yourself paths"
* no universal binary nonsense on OSX

--- configure
+++ configure
@@ -156,24 +156,12 @@
       ;;
     Darwin:*)
       f_platform=macosx-c++
-      if test "$f_insttool" = NO; then
-        f_insttool=/usr/bin/install
-      fi
-      if test -n "`ls /Developer/SDKs/MacOSX10.*u.sdk`"; then
-        f_platform=macosx-uni-c++
-      fi
       ;;
     FreeBSD:*)
       f_platform=freebsd-g++
-      if test "$f_insttool" = NO; then
-        f_insttool=/usr/bin/install
-      fi
       ;;
     HP-UX:*)
       f_platform=hpux-g++
-      if test "$f_insttool" = NO; then
-        f_insttool=/usr/bin/install
-      fi
       ;;
     IRIX64:*)
       f_platform=irix-64
@@ -204,9 +192,6 @@
       ;;
     SunOS:5*)
       f_platform=solaris-g++
-      if test "$f_insttool" = NO; then
-        f_insttool=/usr/bin/install
-      fi
       ;;
     ULTRIX:*)
       f_platform=ultrix-g++
@@ -304,8 +289,8 @@
 
 echo -n "  Checking for GNU make tool... "
 if test "$f_make" = NO; then
-  make_names="gmake make"
-  make_dirs="/usr/bin /usr/local/bin /bin /sbin $bin_dirs"
+  make_names="make"
+  make_dirs="$bin_dirs"
   make_prog=NO
   for i in $make_names; do
     for j in $make_dirs; do
@@ -331,8 +316,8 @@
 
 echo -n "  Checking for GNU install tool... "
 if test "$f_insttool" = NO; then
-  install_names="ginstall install"
-  install_dirs="/usr/bin /usr/local/bin /bin /sbin /usr/ucb $bin_dirs"
+  install_names="install"
+  install_dirs="$bin_dirs"
   install_prog=NO
   install_found=NO
   for i in $install_names; do
@@ -390,7 +375,7 @@
 echo -n "  Checking for perl... "
 if test "$f_perl" = NO; then
   perl_names="perl perl5"
-  perl_dirs="/usr/bin /usr/local/bin /bin /sbin $bin_dirs"
+  perl_dirs="$bin_dirs"
   perl_prog=NO
   perl_found=NO
   for i in $perl_names; do
@@ -500,10 +485,6 @@
   cat >> .tmakeconfig <<EOF
     TMAKE_LFLAGS += -non_shared
 EOF
-  elif test "$f_platform" = "solaris-cc"; then
-  cat >> .tmakeconfig <<EOF
-    TMAKE_LFLAGS += -Bstatic
-EOF
   elif test "$f_platform" = "hpux-cc"; then
   cat >> .tmakeconfig <<EOF
     TMAKE_LFLAGS += -noshared
